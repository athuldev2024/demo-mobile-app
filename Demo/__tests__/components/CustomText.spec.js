import React from 'react';
import {render} from '@testing-library/react-native';
import {Header} from '../../src/components/common/CustomText'; // Ensure Header is exported properly

describe('Header Component', () => {
  it('matches snapshot', () => {
    const {toJSON} = render(<Header>Text</Header>);
    expect(toJSON()).toMatchSnapshot();
  });
});
