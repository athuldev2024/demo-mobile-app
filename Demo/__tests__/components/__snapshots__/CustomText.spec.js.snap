// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component matches snapshot 1`] = `
<Text
  style={
    [
      {
        "textAlign": "left",
      },
      {
        "color": "rgba(28, 27, 31, 1)",
        "writingDirection": "ltr",
      },
      [
        {
          "fontFamily": "System",
          "fontSize": 32,
          "fontWeight": "400",
          "letterSpacing": 0,
          "lineHeight": 40,
        },
        {
          "color": "#6c50a9",
          "fontFamily": "sans-serif-thin",
          "fontWeight": "bold",
        },
      ],
    ]
  }
>
  Text
</Text>
`;
